src/components/Avatar/Avatar.module.css

```diff
@@ -51,6 +58,27 @@
   width: var(--eds-size-6);
 }
 
+.avatar--xl {
+  font: var(--eds-theme-typography-title-lg);
+
+  height: var(--eds-size-8);
+  width: var(--eds-size-8);
+}
+
+.avatar--xxl {
+  font: var(--eds-theme-typography-title-lg);
+
+  height: var(--eds-size-12);
+  width: var(--eds-size-12);
+}
+
+.avatar--xxxl {
+  font: var(--eds-theme-typography-title-lg);
+
+  height: calc(var(--eds-size-base-unit) * 20);
+  width: calc(var(--eds-size-base-unit) * 20);
```

Comment:
Is this just because there is no `--eds-size-20`?

Reply:
correct. I didn't want to create a token for that yet since we might move to t-shirt sizes instead of fixed constants